### ISSUE TITLE
Mongo DB is not loaded and returns Class "MongoDB\Driver\Manager" not found

### ISSUE THREAD

**Issue by AngelAlexQC** [vercel-community/php#548]
# Bug report

- Version: 0.7.1
- URL: Yes https://inti-codes.vercel.app/
- Repository: No

## Description

Im trying setup a laravel application with mongo, but i have problems with vercel deploy because the mongo extension is not loaded and then not works in application
![image](https://github.com/vercel-community/php/assets/9358510/2d4c6681-82d8-42e2-b594-34f73e4c4288)
![image](https://github.com/vercel-community/php/assets/9358510/ca6e712e-e30b-4c84-a1fb-ef6ecce4b408)

**Comment by BintangDeveloper** [comment]
I had the same problem, but I didn't have a solution, so I just changed my database to MySQL.

**Comment by cjogloc** [comment]
I have the same issue. @AngelAlexQC I see your website is working now. How did you get this working?

**Comment by q1an1x** [comment]
> I have the same issue. @AngelAlexQC I see your website is working now. How did you get this working?

Just stumbled upon this and apparently the mongodb extension along with some other extensions have been missing since 0.7.0(PHP8.3) (#525), for now I'm downgrading to 0.6.2(PHP8.2).

**Comment by AngelAlexQC** [comment]
im using this config in `vercel.json`
```json
{
    "version": 2,
    "builds": [
        { "src": "/api/index.php", "use": "vercel-php@0.6.2" },
        { "src": "/public/**", "use": "@vercel/static" }
    ],
    "routes": [
        {
            "src": "/(css|js)/(.*)",
            "dest": "public/$1/$2"
        },
        {
            "src": "/(favicon\\.ico|robots\\.txt)",
            "dest": "/public/$1"
        },
        {
            "src": "/build/(.*)",
            "dest": "/public/build/$1"
        },
        {
            "src": "/img/(.*)",
            "dest": "/public/img/$1"
        },
        {
            "src": "/(.*)",
            "dest": "/api/index.php"
        },
        {
            "src": "/img/(.*)",
            "dest": "/public/img/$1"
        }
    ],
    "env": {
        "APP_NAME": "IntiCodes",
        "APP_ENV": "production",
        "APP_DEBUG": "true",
        "APP_URL": "https://inti.codes",
        "VERCEL_DEMO_MODE": "true",
        "APP_CONFIG_CACHE": "/tmp/config.php",
        "APP_EVENTS_CACHE": "/tmp/events.php",
        "APP_PACKAGES_CACHE": "/tmp/packages.php",
        "APP_ROUTES_CACHE": "/tmp/routes.php",
        "APP_SERVICES_CACHE": "/tmp/services.php",
        "CACHE_DRIVER": "array",
        "LOG_CHANNEL": "stderr",
        "SESSION_DRIVER": "array",
        "VIEW_COMPILED_PATH": "/tmp/views",
        "SSR_TEMP_PATH": "/tmp/ssr",
        "NODE_PATH": "node"
    }
}
```

in vercel im using node v18, in v20 dont works

**Comment by f3l1x** [comment]
MongoDB is supported (fixed) in 0.7.3.
